com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/monitoring/FlowableThreadMonitor.java

```diff
@@ -0,0 +1,95 @@
+package com.sap.cloud.lm.sl.cf.web.monitoring;
+
+import java.lang.Thread.State;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sap.cloud.lm.sl.cf.core.util.ApplicationConfiguration;
+
+@Named
+public class FlowableThreadMonitor implements Runnable {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FlowableThreadMonitor.class);
+
+    private static final String JOB_EXECUTOR_PREFIX = "flowable-async-job-executor-thread-";
+    private static final String ASYNC_EXECUTOR_PREFIX = "asyncExecutor-";
+    private static final String FLOWABLE_MONITOR_THREAD_NAME = "flowable-thread-monitor";
+
+    private static final String ASYNC_EXECUTOR_RUNNING = "asyncExecutorRunning";
+    private static final String ASYNC_EXECUTOR_TOTAL = "asyncExecutorTotal";
+    private static final String JOB_EXECUTOR_RUNNING = "jobExecutorRunning";
+    private static final String JOB_EXECUTOR_TOTAL = "jobExecutorTotal";
+
+    private final Map<String, Integer> threadStatisticsMap = new ConcurrentHashMap<>(4);
```

Comment:
Why do we need a map to store these values? Can we store them in 4 separate variables?